components/global/product_switcher/product_switcher.tsx

```diff
@@ -102,16 +130,35 @@ const ProductSwitcher = (): JSX.Element => {
     });
 
     return (
-        <div ref={menuRef}>
-            <IconButton
-                icon={'products'}
-                onClick={handleClick}
-                size={'sm'}
-                toggled={switcherOpen}
-                inverted={true}
-            />
-            <SwitcherMenu
-                open={switcherOpen}
+        <MenuWrapper
+            open={switcherOpen}
+        >
+            <ProductSwitcherContainer>
+                <IconButton
+                    icon={'products'}
+                    onClick={handleClick}
+                    size={'sm'}
+                    compact={true}
+                    toggled={switcherOpen}
+                    inverted={true}
+                    aria-label='Select to open product switch menu.'
+                />
+                <ProductBranding>
```

Comment:
This should only show for channels right?

Reply:
No I think when we get the other products we will remove the hardcoded values and do this for all of them.

Reply:
Once we have a functionally integrated plugin for one of the other suites, we will make necessary updates to exactly how this (and other areas) works.

Reply:
@deanwhillier We do. Boards is done, just not on community yet. 
@nevyangelova Maybe we shouldn't break it by adding this now? As written this will show up even though boards or playbooks is active (unless I am missing something).